lib/Basics/debugging.cpp

```diff
@@ -118,6 +123,9 @@ void TRI_AddFailurePointDebugging(std::string_view value) {
   {
     WRITE_LOCKER(writeLocker, ::failurePointsLock);
     added = ::failurePoints.emplace(value).second;
+    if (added) {
```

Comment:
I think this `if` condition can be removed here.
If we emplace and the failure point is already present in `::failurePoints`, we could still set the atomic variable to `true`. This would still be correct. And probably the case of adding an already set failure point is unlikely.